### ISSUE TITLE
Error when no results

### ISSUE THREAD

**Issue by blackswanny** [ahmadnassri/node-nightwatch-accessibility#14]
I have markup, which `aXe` `cli` considers valid. No errors. However, when using with nightwatch accessibility plugin the test fails due to no results returned by Axe code. Which is currently considered as error in code of this plugin

**Comment by ahmadnassri** [comment]
can you please provide some html examples to review / test?

**Comment by blackswanny** [comment]
yes, you can find the html here

https://baseui.design/iframe.html?selectedKind=Select&selectedStory=Select&full=0&addons=1&stories=1&panelRight=0&addonPanel=REACT_STORYBOOK%2Freadme%2Fpanel

and the code of test can be found here
https://github.com/uber-web/baseui/issues/424

```js
  it('passes basic a11y tests', browser => {
    goToUrl({
      suite,
      test: scenarios.SELECT,
      browser,
    })
      .initAccessibility()
      .waitForElementVisible(selectors.selectInput, 1000)
      .assert.accessibility('html', {
        rules: {
          'aria-labelledby': {
            enabled: false,
          },
        },
      });
  });
```

**Comment by ahmadnassri** [comment]
the latest release (`v1.10.0`) should have this addressed
